### ISSUE TITLE
MacOS accessibility request is unreliable

### ISSUE THREAD

**Issue by TheLastProject** [Pext/Pext#141]
Somehow, the requests Pext makes to get accessibility access through the "accessibility" module on pypi don't quite always work right, causing trouble with the focus workaround and autotype functionality.

I don't understand why this happens. Help would be amazing.

**Comment by TheLastProject** [comment]
It should be noted that, when removing Pext from accessibility access in macOS' system preferences (by pressing `-`), and then restarting Pext and allowing it access, this issue is resolved. Sadly, only temporarily.

**Comment by TheLastProject** [comment]
This also seems to cause macOS to "freeze up" for a few seconds upon launching Pext

**Comment by TheLastProject** [comment]
I ended up removing the accessibility package altogether.

**Comment by TheLastProject** [comment]
c6e68cdf374e983cce1ebbf8689c0d1e03e29489
